image/Makefile

```diff
@@ -44,6 +45,7 @@ mkosi.output.%/fedora~37/image.raw: mkosi.files/mkosi.%.conf inject-bins inject-
 		$(AUTOLOGIN_ARGS) \
 		--environment=INSTALL_DEBUGD \
 		--environment=CONSOLE_MOTD \
+		--kernel-command-line="$(KERNEL_DEBUG_CMDLNE)=$(DEBUG)" \
```

Comment:
If i understand correctly, this always sets `constellation.debug`, even for release images?

Reply:
atm this is the case, `KERNEL_DEBUG_CMDLNE` is either `true` or `false`. I did this because I thought that would make the parsing easier. If it is false, no debug output will be enabled.

Do you think we should do this differently?

Reply:
I think it would pollute the cmdline less if we could leave it out by default and only set it if debugging should be enabled.
I think we have examples in this makefile on how to implement optional args:
```
AUTOLOGIN_ARGS := $(if $(filter true,$(AUTOLOGIN)),--autologin) # set "--autologin" if AUTOLOGIN is true
```